app/Enums/ServerType.php

```diff
@@ -0,0 +1,11 @@
+<?php
+
+namespace App\Enums;
+
+enum ServerType:string
+{
+    case LOCAL = 'local';
+    case QA = 'qa';
+    case DEMO = 'demo';
+    case PRODUCTION = 'production';
```

Comment:
why do we have these values that look like environments, not server types?

server types so far would be:

- local
- remote-ssh